src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -3263,7 +3301,7 @@ export class ResultsViewPageStore {
     //this is only required to show study name and description on the results page
     //CancerStudy objects for all the cohortIds
     readonly queriedStudies = remoteData({
-        await: () => [this.studyIdToStudy, this.queriedVirtualStudies],
+        await: () => [this.studyIdToStudy, this.queriedVirtualStudies, this.isThereRemoteNGCHM],
```

Comment:
this will introduce a hard dependency on the mdanderson site for ALL portals.  if it errors, or is slow, then whatever feature depends on queriesStudies will fail.    i would make sure that ONLY the link/button depends on remoteNgchmUrl and that it is hidden if it fails.     you will have to deal with loading and errors state then.  don't worry about loading state.  just hidden if it's loading.  for error state, just put in some little error text.

Reply:
Removed the hard dependency.  (Still on RC for now.)